tests/tt_eager/python_api_testing/unit_testing/test_sharded.py

```diff
@@ -528,11 +528,12 @@ def test_sharded_program_cache(device, use_program_cache, function_level_default
 
 
 @pytest.mark.parametrize("in0_sharded", [True, False], ids=["in0_sharded", "in0_unsharded"])
-@pytest.mark.parametrize("out_sharded", [True, False], ids=["out_sharded", "out_unsharded"])
+# @pytest.mark.parametrize("out_sharded", [True, False], ids=["out_sharded", "out_unsharded"])
```

Comment:
why remove testing of `out_sharded = true`?

Reply:
My mistake, didn't mean to push that.